test-kustomize-build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
I'd add:

```suggestion
set -e
```

Otherwise I have no idea which one of the builds in the for loop failed, because stderr goes to `/dev/null` ... Or you can have some conditional `echo` or something...